group_vars/dev

```diff
@@ -74,7 +74,7 @@ python_gdata_rpm_name: python-gdata-2.0.11-1.el6.noarch.rpm
 python_gdata_rpm_url: "{{bahmni_repo_url}}/{{python_gdata_rpm_name}}"
 python_gdata_rpm_location: "{{download_folder}}/{{ python_gdata_rpm_name}}"
 
-postgres_repo_rpm_name: pgdg-centos96-9.6-3.noarch.rpm
+postgres_repo_rpm_name: pgdg-redhat-repo-42.0-11.noarch.rpm
```

Comment:
where are we mentioning what version of pg we are installing? the above seems to the repo for the postgres for all versions